CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Versions
 
 ## x.x.x
+* Add `icon`, `isOptionsViewAvailable`, and `isMediaViewAvailable` to ad object of native ad UI callbacks.
```

Comment:
Can we just combine this into `* Add `title`, `advertiser`, `body`, and `callToAction` to ad object of native ad UI callbacks.`?